### ISSUE TITLE
Port request (Atarist Dcastaway) opendingux

### ISSUE THREAD

**Issue by jutleys** [steward-fu/gh_retrogame_emulator#128]
http://dl.openhandhelds.org/cgi-bin/dingoo.cgi?0,0,0,0,72,535

**Comment by tackro** [comment]
indeed, nice emu

**Comment by steward-fu** [comment]
Ported to RetroGame.
kindly check it
https://github.com/steward-fu/gh_retrogame_binary

**Comment by jutleys** [comment]
Same problem steward will only run via explorer and if i click on the file
dcastaway.

**Comment by jutleys** [comment]
I looks like all the games i am adding will not start from gmenu2x only
with explorer not sure whats wrong but all the games run well thank you.

**Comment by jutleys** [comment]
Ok tested again Reminiscence running perfect though this is a game not emulator handy has loud sound but works fine.Now dcastaway works great controls are fine many games run must be zipped or unzipped in the ST format to run just select game then run.You have trigger buttons that call the mouse and keyboar.The only thing i noticed intro sound on games is a bit messed up but in game sound is fine but great emulator.

**Comment by steward-fu** [comment]
For handy emulator, is "menu = backlight" key in your test ? or "menu = L button" ?

**Comment by jutleys** [comment]
i will check

**Comment by jutleys** [comment]
yes i just tested menu = backlight will exit emulator and then menu = L button will bring up the menu. Sound is too loud.

**Comment by steward-fu** [comment]
the version of "menu = backlight" -> sound ok ?

**Comment by jutleys** [comment]
sound is ok but too loud.
